test/chainsaw/testmetrics/metrics-keda/chainsaw-test.yaml

```diff
@@ -42,18 +52,17 @@ spec:
         - describe:
             resource: keptnmetricsprovider
             name: my-provider
-        - describe:
-            resource: pods
-            selector: app=podtato-head
         - describe:
             resource: scaledobject
             name: test-scaledobject
-      finally:
+    # remove keda in regular step, since in case of failure, we want to have the logs available
```

Comment:
I do not understand this change ... a finally is run independently from if the catch happens or not, if the catch runs you should see the logs anyhow

Reply:
the reason is that we want to have the resources available in the integration test logs (the snapshot of the cluster with all the resources store there). The chainsaw logs are very good for debugging small issues, but when it's not clear, it's better to have a deep view into the cluster

Reply:
I see no difference between the output in chainsaw and the overall logs since the information you get is the same. I would either 100% embrace the chainsaw paradigm or not, somewhere we had a ticket to get rid of integration logs to begin with 🤷‍♀️ but no strong feelings about this

Reply:
i agree with anna here. the finally step will be run after all the logs were gathered anyways